packages/valory/skills/abstract_round_abci/base.py

```diff
@@ -2531,6 +2536,130 @@ def cleanup_current_histories(self, cleanup_history_depth_current: int) -> None:
         )
 
 
+class OffenseType(Enum):
+    """The types of offenses."""
```

Comment:
A warning about the value to seriousness could go here, we can do it in the next pr.